README.md

```diff
@@ -69,15 +69,22 @@ cmake -G Ninja llvm/llvm -B build \
     -DLLVM_ENABLE_PROJECTS=mlir \
     -DLLVM_EXTERNAL_PROJECTS=circt \
     -DLLVM_EXTERNAL_CIRCT_SOURCE_DIR=$PWD
+```
 
-# Run the build
+If you want to build everything about the CIRCT tools and libraries, run below command(also runs all tests):
+```
 ninja -C build check-circt
 ```
 
-The above builds the CIRCT tools and libraries and runs all regression tests.
-If you want to include `circt-verilog` in the build, add `-DCIRCT_SLANG_FRONTEND_ENABLED=ON` to the cmake call.
-You can ask ninja to only build a specific library or tool, such as `ninja -C build circt-opt`.
-This will only build the necessary parts of LLVM, MLIR, and CIRCT, which can be a lot quicker than building everything.
+If you want to only build a specific part, for example the `circt-opt` tool(only build the necessary parts of LLVM, MLIR, and CIRCT, a lot quicker):
+```sh
+ninja -C build circt-opt
+```
+
+If you only want to build the `firtool` binary:
+```sh
+ninja -C build bin/firtool
```

Comment:
it's a little inconsistent to have firtool sit in bin and not circt-opt no? Either both should be in bin or neither imo

Reply:
please fix this